_docs/welcome/feature-stages.md

```diff
@@ -7,6 +7,7 @@ order: 10
 layout: docs
 type: markdown
 redirect_from: "/docs/reference/release-roadmap.html"
+redirect_from: "/docs/reference/feature-stages.html"
```

Comment:
try:
```
redirect_from:
  - "/docs/reference/release-roadmap.html"
  - "/docs/reference/feature-stages.html"
```
because currently, it seems it's last one wins and it loses the current redirect